### PR TITLE
feat: InlineWysiwyg default image upload function

### PR DESCRIPTION
**Pull request by ncphillips**

### CODE REVIEW

File: packages/react-tinacms-editor/README.md [hunk 1]
```diff
@@ -135,7 +136,27 @@ export function Page(props) {
 }
 ```
 
-### Example: Dynamic Imports
+### Sticky Menu
+
+When editing long content it is likely that the editor will scroll past the wysiwyg menu.
+
+By adding the `sticky` property the menu will follow the user as they scroll through the page.
```
Comment (DirtyF) [inline packages/react-tinacms-editor/README.md:143]: According to the docs sticky is the default, right?

**Comment by DirtyF** [comment]
Just tried to test on the demo-gatsby package and got an image/undefined error after drag and dropping the image:

tinacms 2020-08-06 à 20 59 29

File: packages/react-tinacms-editor/README.md (hunk 1 above)
Comment (ncphillips) [reply packages/react-tinacms-editor/README.md:143]: Nope, default is non-sticky

**Comment by ncphillips** [comment]
I can't reproduce that 🤔

File: packages/react-tinacms-editor/src/tinacms-inline/inline-wysiwyg.tsx [hunk 2]
```diff
@@ -33,10 +33,27 @@ export function InlineWysiwyg({
...
+            directory: passedInImageProps?.directory || '',
+            file,
+          }))
+        )
+
+        return allMedia.map(media => `${media.directory}${media.filename}`)
+      },
+      ...passedInImageProps,
+    }
+  }, [cms.media.store])
```
Comment (dwalkr) [inline packages/react-tinacms-editor/src/tinacms-inline/inline-wysiwyg.tsx:55]:
```suggestion
  }, [cms.media.store, passedInImageProps])
```
should we add this here?

**Comment by DirtyF** [comment]
@ncphillips OK I get what happened here, when you try to upload an image that already exists, you get the undefined.

https://www.loom.com/share/98717e9589a64689908402a685e6ed30

**Comment by ncphillips** [comment]
Oh! That's interesting. Unrelated to this PR but let's make note

File: packages/react-tinacms-editor/src/tinacms-inline/inline-wysiwyg.tsx (hunk 2 above)
Comment (ncphillips) [reply packages/react-tinacms-editor/src/tinacms-inline/inline-wysiwyg.tsx:55]: Maybe. It would probably make the hook rerun on every render

Comment (dwalkr) [reply packages/react-tinacms-editor/src/tinacms-inline/inline-wysiwyg.tsx:55]: maybe we're memoizing the wrong thing then

```ts
const defaultUpload = React.useCallback((files: File[]) => {
  //...
}, [cms.media.store])

const imageProps = {
  upload: defaultUpload,
  ...passedInImageProps
}
```

**Pull request merged**